cmd/nri-vsphere/nri-vsphere.go

```diff
@@ -45,12 +46,22 @@ func main() {
 		config.Logrus.WithError(err).Fatal("failed to create client")
 	}
 
+	config.VMWareClientRest, err = client.NewRest(config.VMWareClient, config.Args.User, config.Args.Pass)
```

Comment:
We create the rest client and the tags manager event if the tags are not collected, should add a check?
There is not a real drawback from creating the client and the manager, but it is not needed